src/DocsParser.ts

```diff
@@ -146,6 +146,18 @@ export class DocsParser {
           'HTMLElement documentation should not be considered a class',
         );
       }
+      let electronProcess = { main: false, renderer: false };
```

Comment:
Now that this isn't being overwritten it should be const

```suggestion
      const electronProcess = { main: false, renderer: false };
```